CommMicro.py

```diff
@@ -11,41 +11,40 @@
 add cm # to filename with -F switch
 """
 
-import sys
-from pydm import Display
-from PyQt5.QtGui import QStandardItem, QPixmap, QScreen
-from PyQt5.QtWidgets import (QWidgetItem, QCheckBox, QPushButton, QLineEdit,
-                             QGroupBox, QHBoxLayout, QMessageBox, QWidget,
-          QApplication,QLabel, QFrame, QComboBox, QRadioButton,QFileDialog)
-from os import path, pardir, makedirs, system, listdir
+# for ceil
 import subprocess
-#from qtpy.QtCore import Slot, QTimer
-from functools import partial, reduce
-from datetime import datetime, timedelta
 import sys
+from datetime import datetime
+from functools import partial
+from os import makedirs, path
+
 import numpy as np
-from scipy.fftpack import fft, fftfreq
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import (QFileDialog, QWidget)
 from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg
-import matplotlib.pyplot as plt
 from matplotlib.figure import Figure
-
-# for ceil
-import math
+from pydm import Display
+from qtpy.QtCore import Slot
+from scipy.fftpack import fft, fftfreq
 
 # FFt_math has utility functions
 import FFt_math
 
-LASTPATH=''
-DEBUG=1
+BUFFER_LENGTH = 16384
+DEFAULT_SAMPLING_RATE = 2000
+
+LASTPATH = ''
+DEBUG = 1
 DATA_DIR_PATH = "/u1/lcls/physics/rf_lcls2/microphonics/"
 
+
 class MplCanvas(FigureCanvasQTAgg):
-# MPLCanvas is the class for the 'canvas' that plots are drawn on and then mapped to the ui
-# They are Figure format described in matplotlib 2.2 documentation
+    # MPLCanvas is the class for the 'canvas' that plots are drawn on and then mapped to the ui

```

Comment:
I would say move `#` comments on top of what they're annotating or convert them to `""" _ """` syntax

Reply:
done